test/Project.toml

```diff
@@ -1,27 +1,20 @@
 [deps]
-Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
 Aqua = "4c88cf16-eb10-579e-8560-4a9242c79595"
 ArgParse = "c7e460c6-2fb9-53a9-8c5b-16f535851c63"
 Artifacts = "56f22d72-fd6d-98f1-02f0-08ddc0907c33"
 CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
-ClimaAnalysis = "29b5916a-a76c-4e73-9657-3c8fd22e65e6"
 ClimaAtmos = "b2c96348-7fb7-4fe0-8da9-78d88439e717"
 ClimaComms = "3a4d1b5c-c61d-41fd-a00a-5873ba7a1b0d"
 ClimaCore = "d414da3d-4745-48bb-8d80-42e94e092884"
 ClimaCoreMakie = "908f55d8-4145-4867-9c14-5dad1a479e4d"
 ClimaCoupler = "4ade58fe-a8da-486c-bd89-46df092ec0c7"
-ClimaLand = "08f4d4ce-cf43-44bb-ad95-9d2d5f413532"
 ClimaParams = "5c42b081-d73a-476f-9059-fd94b934656c"
 ClimaTimeSteppers = "595c0a79-7f3d-439a-bc5a-b232dc3bde79"
```

Comment:
From your comment in #1075, it looks like ClimaTimeSteppers, IntervalSets, LinearAlgebra, SciMLBase, and Statistics can also be removed. These are indirect dependencies, so if you're planning to do them in a separate PR that's fine, but I see that this PR removes the indirect dependencies in the `experiments/ClimaEarth/` environment, so maybe you could do them all at once. Whichever is easiest for you!

Reply:
My comment in #1075 was incorrect about a few things. 

For `experiments/ClimaEarth`, Interpolations.jl cannot be removed as a direct dependency because it is used once in `plot_helper.jl`. 

For `test`, ClimaTimeSteppers, IntervalSets, LinearAlgebra, SciMLBase, and Statistics cannot be removed as direct dependencies, because they are imported in scripts in `experiments/ClimaEarth`, which are then `include`-ed in tests in `test`.

I've updated the comment reflect this

Reply:
IntervalSets can be often removed with small changes

Reply:
IntervalSets now removed.